sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile

```diff
@@ -100,34 +99,48 @@ create-site: instance/etc/zope.ini ## Create a new site from scratch
 
 # Example Content
 .PHONY: update-example-content
-update-example-content: $(BIN_FOLDER)/tox ## Export example content inside package
+update-example-content: $(BIN_FOLDER)/uv ## Export example content inside package
 	@echo "$(GREEN)==> Export example content into $(EXAMPLE_CONTENT_FOLDER) $(RESET)"
 	if [ -d $(EXAMPLE_CONTENT_FOLDER)/content ]; then rm -r $(EXAMPLE_CONTENT_FOLDER)/* ;fi
 	$(BIN_FOLDER)/plone-exporter instance/etc/zope.conf $(PLONE_SITE_ID) $(EXAMPLE_CONTENT_FOLDER)
 
-.PHONY: check
-check: $(BIN_FOLDER)/tox ## Check and fix code base according to Plone standards
+# QA
```

Comment:
@davisagli oh, I see, we splitted it in two, then we can use them both with check. Then the above comment is still true, since the CI do not format, only lints.

Reply:
I agree, it's surprising that "check" would include formatting.

Reply:
I just kept check because we used that before

Reply:
"because we used that before" is not a good reason for continuing to use a badly named thing. Why can't we correct it?

Reply:
@stevepiercy What do you suggest for the command that does both lint and format?

Reply:
I've heard the term "static analysis" used to include these functions, but we should resist the temptation to shorten that to `stat-anal`.

We went with separate commands of [`lint` and `format` in Volto](https://docs.google.com/spreadsheets/d/1C_kBeXrx5nRLYaLI8qNXhBOBD9j7Ty0r_DxJNDz7Cww/edit?gid=0#gid=0). We did not combine them into a single command. I think I would prefer to keep them separate, then run `make lint format` when I want to run them both with a single command. That would also allow me to run only either `make lint` or `make format` when I want to focus on a single aspect.